packages/block-editor/src/components/url-popover/link-viewer.js

```diff
@@ -35,6 +38,7 @@ function LinkViewerUrl( { url, urlLabel, className } ) {
 
 export default function LinkViewer( {
 	className,
+	isFullWidth,
```

Comment:
Is this property going to be very common? Can not media-replace-flow/style.scss and the width: 100% based on the block-editor-url-popover__link-viewer-url className?

Reply:
I felt like this was a mistake to not be present: as long as the component is placeable in all kinds of containers, it should have the full width option ready to set. It's the same reasoning for why URLInput has this property, the viewer, like the input needs it as well.